app/presenters/html_publication_presenter.rb

```diff
@@ -31,4 +31,12 @@ def last_changed
   def organisations
     content_item["links"]["organisations"].sort_by { |o| o["title"] }
   end
+
+  def organisation_logo(organisation)
+    super(organisation).tap do |logo|
```

Comment:
You don't need to pass the `organisation` argument here. It happens automagically.